core/platform_feature_list.py

```diff
@@ -24,7 +24,7 @@
 
 ParamNames = params.ParamNames
 
-# Names of feature objects defined in domain/platform_parameter_list.py
+# Names of feature objects defined in domain/params.py
```

Comment:
Does this file actually exist?

Reply:
Oh, I think it got changed while I doing "search-and-replace", thanks for noticing it. I have updated it.